services/web-app/stories/mdx/divider.stories.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Column, Grid } from '@carbon/react'
+
+import Divider from '@/components/divider'
+
+const stories = {
+  title: 'MDX/Divider',
+  component: Divider
+}
```

Comment:
Let's hide these controls, they don't make much sense? 
<img width="1240" alt="image" src="https://user-images.githubusercontent.com/40550942/173887863-23b23b43-0bfd-407e-9ab0-ff1ed893108e.png">

```suggestion
  title: 'MDX/Divider',
  component: Divider,,
  argTypes: {
    children: {
      control: false
    },
    className: {
      control: false
    }
  }
}
```